docs/Manuals/Server/options.rst

```diff
@@ -0,0 +1,1176 @@
+
+  Possible values (option can take any number of these):
+
+    * ``WRAPX``: Wrap East-West
+    * ``WRAPY``: Wrap North-South
+    * ``ISO``: Isometric
+    * ``HEX``: Hexagonal
+
+``trade_revenue_style``
+  :strong:`Value (Min, Max)`: Classic Freeciv (``CLASSIC``)
+
+  :strong:`Description`: The formula for the trade a city receives from a traderoute. ``CLASSIC`` revenues
+  depend on distance and trade with multipliers for overseas and international routes. ``SIMPLE`` revenues are
+  proportional to the average trade of the two cities.
+
+``trademindist``
+  :strong:`Value (Min, Max)`: 9 (1, 999)
+
+  :strong:`Description`: Minimum distance (tiles) for trade routes. In order for two cities in the same
+  civilization to establish a trade route, they must be at least this far apart on the map. For square grids,
+  the distance is calculated as *Manhattan distance*, that is, the sum of the displacements along the
+  :math:`x` and :math:`y` directions.
+
+``tradeworldrelpct``
+  :strong:`Value (Min, Max)`: 50 (0, 100)
+
+  :strong:`Description`: How largely trade distance is relative to world size. When determining trade between
+  cities, the distance factor can be partly or fully relative to world size. This setting determines how big
+  percentage of the bonus calculation is relative to world size, and how much only absolute distance matters.
+
+``trading_city``
+  :strong:`Value (Min, Max)`: enabled
+
+  :strong:`Description`: City trading via treaty. If turned off, trading cities in the diplomacy dialog is not
+  allowed.
+
+``trading_gold``
+  :strong:`Value (Min, Max)`: enabled
+
+  :strong:`Description`: Gold trading via treaty. If turned off, trading gold in the diplomacy dialog is not
+  allowed.
+
+``trading_tech``
+  :strong:`Value (Min, Max)`: enabled
+
+  :strong:`Description`: Technology trading via treaty. If turned off, trading technologies in the diplomacy
+  dialog is not allowed.
+
+``traitdistribution``
+  :strong:`Value (Min, Max)`: Fixed (``FIXED``)
+
+  :strong:`Description`: :term:`AI` trait distribution method. Possible values:
+
+    * ``FIXED``: Fixed
+    * ``EVEN``: Even
+
+``turnblock``
+  :strong:`Value (Min, Max)`: enabled
+
+  :strong:`Description`: Turn-blocking game play mode. If this is turned on, the game turn is not advanced
+  until all players have finished their turn, including disconnected players.
+
+``unitwaittime``
+  :strong:`Value (Min, Max)`: 0 (0, 8639999)
+
+  :strong:`Description`: Minimum time between unit actions over turn change. This setting gives the minimum
+  amount of time in seconds between unit moves and other significant actions (such as building cities) after a
+  turn change occurs. For example, if this setting is set to 20 and a unit moves 5 seconds before the turn
+  change, it will not be able to move or act in the next turn for at least 15 seconds. This value is limited
+  to a maximum value of two-thirds of ``timeout``. ``unitwaittime`` (aka UWT) is a tool to help reduce
```

Comment:
Do we have UWT in the glossary? We probably should, it's a fairly commonly used term, especially in LT games. This should then link to it.